sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/display/DisplayDataMatchers.java

```diff
@@ -57,6 +57,14 @@ private DisplayDataMatchers() {}
   }
 
   /**
+   * Creates a matcher that matches if the examined {@link DisplayData} contains an item with the
+   * specified key.
+   */
+  public static Matcher<DisplayData> hasDisplayItem(String key) {
```

Comment:
hasDisplayKey, a la [Matchers#hasKey](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#hasKey(K))

The other methods would, for consistency, be hasDisplayEntry, but that can be done in a different PR.